process/block/metablock.go

```diff
@@ -547,6 +547,11 @@ func (mp *metaProcessor) CommitBlock(
 
 	mp.appStatusHandler.SetStringValue(core.MetricCurrentBlockHash, core.ToB64(headerHash))
 
+	pubKeys, err := mp.nodesCoordinator.GetValidatorsPublicKeys(header.PrevRandSeed, header.Round, sharding.MetachainShardId)
```

Comment:
I would move the metrics handling into another function so we won't crowd this one once new metrics will be handled

Reply:
Done. Metrics was moved in another function.